### ISSUE TITLE
Download Chart button doesn't work in IE

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3097]
The ‘Download Chart CSV’ link does not work in IE 11. However, when browsing in Google Chrome or Mozilla Firefox, users are able to download the chart csv without issue.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [] Windows: IE11
- [] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by LucyGwilliamAdmin** [comment]
I'm closing this as it is an Open SDG issue. have added issue in Open SDG: https://github.com/open-sdg/open-sdg/issues/313
